setup.sh

```diff
@@ -35,7 +35,7 @@ function check_installed_python() {
             return
         fi
     done
-
+    PYTHON="python3.10"
```

Comment:
Why add this here? it seems pointless - we exit 2 lines after this ...

Reply:
If no python versions are installed, it will install python3.8 as it is last processed.

Reply:
Freqtrade will install no python version itself. 2 lines after this, the script will exit if no python version is installed.